test/fragment.ts

```diff
@@ -0,0 +1,57 @@
+import { describe, it } from "node:test";
+import { strictEqual as equal } from "node:assert";
+import expand, { UserConfig } from "../src";
+
+describe("frag", () => {
+    const config: UserConfig = { options: { "jsx.enabled": true } };
+
+    it("should expand fragment", () => {
+        // should it work without explicit jsx.enabled ? It's not a valid html tag afaik
```

Comment:
Here is the question i had. Should the result actually be expanded to ` <></>` ?

Reply:
I think it’s OK to expand to `<>\n\t${0}\n</>` since its use case is to insert multiple nodes which will be formatted anyways